### PR TITLE
chore(examples): Add two redux examples

### PR DESCRIPTION
**Pull request by graysonhicks**
These are two examples of working with Gatsby + Redux that will go along with a WIP blog post.

### CODE REVIEW

File: examples/using-redux-w-interaction-code-splitting/gatsby-browser.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Implement Gatsby's Browser APIs in this file.
+ *
+ * See: https://www.gatsbyjs.com/docs/browser-apis/
+ */
+
+// You can delete this file if you're not using it
+
+export { default as wrapRootElement } from "./src/redux/reduxWrapper"
```
Comment (LekoArts) [inline examples/using-redux-w-interaction-code-splitting/gatsby-browser.js:9]: To have proper hot reloading you'll want to call the `wrapRootElement` api and use a component that is in PascalCase (Fast Refresh doesn't like lowercased components).

See https://github.com/system-ui/theme-ui/blob/develop/examples/gatsby/gatsby-browser.js for an example

File: examples/using-redux-w-interaction-code-splitting/gatsby-ssr.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Implement Gatsby's SSR (Server Side Rendering) APIs in this file.
+ *
+ * See: https://www.gatsbyjs.com/docs/ssr-apis/
+ */
+
+// You can delete this file if you're not using it
+
+export { default as wrapRootElement } from "./src/redux/reduxWrapper"
```
Comment (LekoArts) [inline examples/using-redux-w-interaction-code-splitting/gatsby-ssr.js:9]: Same as `gatsby-browser.js` comment above

File: examples/using-redux-w-interaction-code-splitting/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "dependencies": {
+    "gatsby": "^3.0.0",
+    "gatsby-image": "^3.0.0",
```
Comment (LekoArts) [inline examples/using-redux-w-interaction-code-splitting/package.json:9]: Here are some unused packages in the package.json. image, offline, source-filesystem are all not used

File: examples/using-redux-w-interaction-code-splitting/src/redux/reduxWrapper.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from "react"
+import { Provider } from "react-redux"
+import store from "./store"
+
+export default ({ element }) => (
```
Comment (LekoArts) [inline examples/using-redux-w-interaction-code-splitting/src/redux/reduxWrapper.js:5]: Anonymous functions are bad practice

File: examples/using-redux-w-page-code-splitting/gatsby-node.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Node APIs in this file.
+ *
+ * See: https://www.gatsbyjs.com/docs/node-apis/
+ */
+
+// You can delete this file if you're not using it
```
Comment (LekoArts) [inline examples/using-redux-w-page-code-splitting/gatsby-node.js:7]: Delete empty files

File: examples/using-redux-w-page-code-splitting/package.json [hunk 6]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "dependencies": {
+    "gatsby": "^2.26.1",
+    "gatsby-image": "^2.5.0",
```
Comment (LekoArts) [inline examples/using-redux-w-page-code-splitting/package.json:9]: Some unused packages in here

File: examples/using-redux-w-page-code-splitting/src/redux/reducers/darkMode.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+const TOGGLE_DARKMODE = "TOGGLE_DARKMODE"
+
+export const toggleDarkMode = isDarkMode => ({
+  type: TOGGLE_DARKMODE,
+  isDarkMode,
+})
+
+export default (state = initialState, action) => {
```
Comment (LekoArts) [inline examples/using-redux-w-page-code-splitting/src/redux/reducers/darkMode.js:12]: Anonymous function

File: examples/using-redux-w-page-code-splitting/src/redux/reducers/dummy.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+const LOAD_DATA = "LOAD_DATA"
+
+export const loadData = isLoaded => ({
+  type: LOAD_DATA,
+  isLoaded,
+})
+
+export default (state = initialState, action) => {
```
Comment (LekoArts) [inline examples/using-redux-w-page-code-splitting/src/redux/reducers/dummy.js:17]: Anonymous function

File: examples/using-redux-w-page-code-splitting/src/redux/reduxWrapper.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
+import React from "react"
+import { Provider } from "react-redux"
+import configureStore from "./store"
+
+export default ({ element }) => (
```
Comment (LekoArts) [inline examples/using-redux-w-page-code-splitting/src/redux/reduxWrapper.js:5]: Anonymous function

**Review by LekoArts — CHANGES_REQUESTED** [review]
Thanks! Added some comments

**Comment by graysonhicks** [comment]
Thanks @LekoArts !  Those issues should all be resolved now.

**Review by LekoArts — APPROVED** [review]

**Pull request merged**